contracts/lbp/LBPManager.sol

```diff
@@ -0,0 +1,290 @@
+
+    modifier onlyAdmin() {
+        require(msg.sender == admin, "LBPManager: caller should be admin");
+        _;
+    }
+
+    /**
+     * @dev                             Transfer admin rights.
+     * @param _newAdmin                 Address of the new admin.
+     */
+    function transferAdminRights(address _newAdmin) external onlyAdmin {
+        require(
+            _newAdmin != address(0),
+            "LBPManager: new admin can not be zero"
+        );
+        admin = _newAdmin;
+    }
+
+    /**
+     * @dev                             Initialize LBPManager.
+     * @param _LBPFactory               LBP factory address.
+     * @param _beneficiary              The address that receives the _primeDaoFeePercentage.
+     * @param _name                     Name of the LBP.
+     * @param _symbol                   Symbol of the LBP.
+     * @param _tokenList                Numerically sorted array (ascending) containing two addresses:
+                                            - The address of the project token being distributed.
+                                            - The address of the funding token being exchanged for the project token.
+     * @param _amounts                  Sorted array to match the _tokenList, containing two parameters:
+                                            - The amounts of project token to be added as liquidity to the LBP.
+                                            - The amounts of funding token to be added as liquidity to the LBP.
+     * @param _weights                  Sorted array to match the _tokenList, containing two parametes:
+                                            - The start weight for the project token in the LBP.
+                                            - The start weight for the funding token in the LBP.
+     * @param _startTimeEndTime         Array containing two parameters:
+                                            - Start time for the LBP.
+                                            - End time for the LBP.
+     * @param _endWeights               Sorted array to match the _tokenList, containing two parametes:
+                                            - The end weight for the project token in the LBP.
+                                            - The end weight for the funding token in the LBP.
+     * @param _swapFeePercentage        Percentage of fee paid for every swap in the LBP.
+     * @param _primeDaoFeePercentage    Percentage of fee paid to PrimeDao for providing the service of the LBP Manager.
+     */
+    function initializeLBP(
+        address _LBPFactory,
+        address _beneficiary,
+        string memory _name,
+        string memory _symbol,
+        IERC20[] memory _tokenList,
+        uint256[] memory _amounts,
+        uint256[] memory _weights,
+        uint256[] memory _startTimeEndTime,
+        uint256[] memory _endWeights,
+        uint256 _swapFeePercentage,
```

Comment:
Don't know where to place it so I'll place it here, according to the #117 We need to have a fee which is **changeable** but there is no `SetFee` function or smth. Have this functionality been implemented, or have I misunderstood the issue?

Reply:
See explanation above